Header header
geometry_msgs/Vector3 drone_p
geometry_msgs/Vector3 drone_pdot
geometry_msgs/Vector3 drone_q
geometry_msgs/Vector3 drone_qdot

geometry_msgs/Vector3 boat_p
geometry_msgs/Vector3 boat_pdot
float64 heading

geometry_msgs/Vector3 gps_offset
float64 heading_offset

# ENU Origin coordinate [lon,lat,alt]
geometry_msgs/Vector3 origin
